presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloClient.java

```diff
@@ -175,6 +151,46 @@ public static synchronized Connector getAccumuloConnector(AccumuloConfig config)
     }
 
     /**
+     * Creates and starts an instance of MiniAccumuloCluster, returning the new instance
+     *
+     * @return New MiniAccumuloCluster
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    private static MiniAccumuloCluster createMiniAccumuloCluster()
+            throws IOException, InterruptedException
```

Comment:
Where you expect `InterruptedException` being thrown?
Here you catch it: https://github.com/prestodb/presto/pull/5030/files/65836f8f880c493f2cbb605fd4142d0a67cf54fa..8869b1300b345ff6e4c7cf69929283ddc5968440#diff-68cb92636ee64a829fa0a3578bf988f7R177 ?